pkgs/top-level/all-packages.nix

```diff
@@ -9175,7 +9175,24 @@ with pkgs;
 
   ffcast = callPackage ../tools/X11/ffcast { };
 
-  fflas-ffpack = callPackage ../development/libraries/fflas-ffpack {};
+  fflas-ffpack = callPackage ../development/libraries/fflas-ffpack {
+    # We need to use blas instead of openblas on darwin,
+    # see https://github.com/NixOS/nixpkgs/pull/45013.
+    # We need to pass a different name to --with-blas-libs 
+    # based on which one we use.
+    blas = let addLinkName = { pkg, linkName }:
```

Comment:
You can just add it directly to the openblas and blas packages. `passthru` won't cause a rebuild.

Reply:
Ohh you're saying I should modify them to add the `passthru` attribute in `all-packages.nix`.